src/settingsview.cpp

```diff
@@ -31,14 +33,18 @@ SettingsView::SettingsView(QWidget *parent)
     ui->monitorHelp->setText("<b>?</b>");
     ui->monitorHelp->setToolTip(gt("monitor-directory-tooltip"));
     ui->moveToTrashLabel->setText(gt("move-files-to-trash"));
+    ui->reopenTabLabel->setText(gt("open-previous-tabs-on-startup"));
```

Comment:
I don't remember if entries for `open-previous-tabs-on-startup` have been added in `resources/i18n/en.json` and `resources/i18n/qqq.json` in previous versions of this PR, but currently they are not there.

Reply:
They are not. I thought originally these translations are added with translation wiki PRs after the merge.

Are these two files the only ones for me to change at the moment?

Reply:
Yes. Put the text in English (I thinks @kelson42's choice is "Open previous tabs at startup") in `en.json` and the explanation of that message for translators (since sometimes translations cannot be done out of context) in `qqq.json`.

Reply:
Done